theseus_gui/src/components/ui/Instance.vue

```diff
@@ -12,22 +15,55 @@ const props = defineProps({
     },
   },
 })
+
+const stopBtn = ref(null)
+const playBtn = ref(null)
+
+const uuid = ref(null)
+
+const router = useRouter()
+
+const seeInstance = async () => {
+  const instancePath = `/instance/${encodeURIComponent(props.instance.path)}`
+  await router.push(instancePath)
+}
```

Comment:
this can probably be inlined

Reply:
The e.stopPropagation during the Play click didn't always stop the RouterLink (that existed on the card originally) from navigating. This provided a much more consistent experience.

Reply:
By inlined I mean it can just be a lambda rather than a seperate function